Tools/AP_Periph/can.cpp

```diff
@@ -1781,8 +1781,10 @@ void AP_Periph_FW::esc_telem_update()
             pkt.temperature = nan;
         }
         float rpm;
-        if (esc_telem.get_raw_rpm(i, rpm)) {
+        float error_rate;
+        if (esc_telem.get_raw_rpm_and_error_rate(i, rpm, error_rate)) {
             pkt.rpm = rpm;
+            pkt.error_count = error_rate;
```

Comment:
Wouldn't count be the integral of rate? What's going on here?

Reply:
Yeah, but this means we get back out what we put in with the existing parsing of the message on the vehicle side. The spec does not give a clear description of what it should be.
```
uint32 error_count          # Resets when the motor restarts
```